package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -560,10 +560,15 @@ def _read_next_timestep(self):
 
         triclinic = len(f.readline().split()) == 9  # ITEM BOX BOUNDS
         if triclinic:
-            xlo, xhi, xy = map(float, f.readline().split())
-            ylo, yhi, xz = map(float, f.readline().split())
+            xlo_bound, xhi_bound, xy = map(float, f.readline().split())
```

Comment:
Would it be possible to add a relevant test (maybe based on the issue this aims to fix) that covers these lines?

Reply:
Ah of course, I can write a test that reads in a triclinic box. This would test that it doesnt fail. Testing that the actual issue would be resolved is difficult/impossible?. I think one would have to create a triclinic box in lammps, then actually run lammps and read the output to see if the dimensions are the same.

Reply:
So here that's what we would require to do - create a triclinic box in lammps and then check that we reproduce expected outcomes (calculated by hand and/or comparing to a lammps output).

If we can't do such a correctness check then we might have to discuss further whether or not we can offer this option.

Reply:
I have a first correctness check here:
[triclinic_lammps_2021_09_02.zip](https://github.com/MDAnalysis/mdanalysis/files/7095036/triclinic_lammps_2021_09_02.zip)
I use the values from the input data file for lammps that require xlo, xhi etc. directly and compare those dimensions to the ones imported from the lammpsdump file of the same system:
```python
import MDAnalysis as mda
import numpy as np
from MDAnalysis.lib import mdamath

u = mda.Universe("dump.atom", topology_format='LAMMPSDUMP')

## Direct values from lammps data file
xlo = -0.32115478301032807
xhi = 16.831069399898624
ylo = -0.12372358703610897 
yhi = 25.95896427399614
zlo = -0.045447071698045266 
zhi = 12.993982724334792
xy = 1.506743915478767
xz = - 6.266414551929444 
yz = - 0.42179319547892025

box = np.zeros((3, 3), dtype=np.float64)
box[0] = xhi - xlo, 0.0, 0.0
box[1] = xy, yhi - ylo, 0.0
box[2] = xz, yz, zhi - zlo

direct_dimensions = mdamath.triclinic_box(*box)

print(direct_dimensions == u.dimensions)
```
resulting in the same dimensions
```python
[ True  True  True  True  True  True]
```
Would this be sufficient? Should I then write this procedure into a test file or what would be the best practice for testing specifications set in other software?

Reply:
You can write something like this into the testsuite! the relevant file is [here](https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/coordinates/test_lammps.py).

You will need to learn a bit of pytest, and use the text fixtures but hopefully the file is commented enough to be comprehensible. The [numpy testing functions](https://numpy.org/doc/stable/reference/routines.testing.html) will be very helpful. See [here](https://userguide.mdanalysis.org/stable/testing.html) for how to run the testsuite.


If you need to add an additional file to the test data, please do add it to the `mdanalysis/testsuite/MDAnalysisTests/data/lammps` folder and import it in 
`mdanalysis/testsuite/MDAnalysisTests/datafiles.py` 


I am going to tag @richardjgowers in case I am missing something about `ortho` vs `triclinic` in MDA/LAMMPS here?

Reply:
Ah awesome thank you! I will get the keywords you recommended and then write it up in there!